modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/PartitionMetaTest.java

```diff
@@ -37,7 +37,7 @@
 public class PartitionMetaTest {
     @Test
     void testTreeRootPageId() {
-        PartitionMeta meta = new PartitionMeta(null, 0, 0, 0);
+        PartitionMeta meta = new PartitionMeta(null, 0, 0, 0, 0, 0, 0);
```

Comment:
This constructor starts looking ridiculous, maybe we should use some default constructor and only set fields that we need?

Reply:
fix it